tests/Fixer/PhpUnit/PhpUnitDedicateAssertFixerTest.php

```diff
@@ -344,4 +344,136 @@ public function testInvalidConfig()
 
         $this->fixer->configure(['target' => '_unknown_']);
     }
+
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideTestAssertCountCases
+     */
+    public function testAssertCount($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideTestAssertCountCases()
+    {
+        return [
+            // positive fixing
+            'assert same' => [
+                '<?php $this->assertCount(1, $a);',
+                '<?php $this->assertSame(1, count($a));',
```

Comment:
What about `sizeof`? :thinking:

Reply:
should very easy to add, can someone make an issue/PR?

Reply:
@SpacePossum I'll :)